sumologic/resource_sumologic_connection.go

```diff
@@ -77,9 +77,14 @@ func resourceSumologicConnection() *schema.Resource {
 			"webhook_type": {
 				Type:         schema.TypeString,
 				Optional:     true,
-				ValidateFunc: validation.StringInSlice([]string{"AWSLambda", "Azure", "Datadog", "HipChat", "PagerDuty", "Slack", "Webhook", "NewRelic", "Jira", "Opsgenie", "MicrosoftTeams"}, false),
+				ValidateFunc: validation.StringInSlice([]string{"AWSLambda", "Azure", "Datadog", "HipChat", "PagerDuty", "Slack", "Webhook", "NewRelic", "Jira", "Opsgenie", "MicrosoftTeams", "ServiceNow"}, false),
 				Default:      "Webhook",
 			},
+			"connection_subtype": {
+				Type:         schema.TypeString,
+				Optional:     true,
+				ValidateFunc: validation.StringInSlice([]string{"Incident", "Event", ""}, false),
```

Comment:
I wonder what's the best practice around writing empty arguments (`""`) in plans vs omitting them 🤔 
Would it break anything if we disallowed empty `connection_subtype`?

Reply:
ah yup empty string would be invalid on server-side - I think I added the empty string while trying to figure out why the tests are failing (it was a different issue), will remove it